tests/nested/core/hotplug/task.yaml

```
@@ -71,13 +66,7 @@ execute: |
     hotplug_del_dev1
 
     # sanity check to make sure qemu event was triggered correctly
-    for _ in $(seq 5); do
-        if tests.nested exec "udevadm info -e" | MATCH "ID_MODEL=QEMU_USB_SERIAL"; then
-            sleep 1
-        else
-            break
-        fi
-    done
+    tests.nested exec "retry --wait 1 -n 5 sh -c 'udevadm info -e | NOMATCH ID_MODEL=QEMU_USB_SERIAL'"
```

this one could also be changed and then only has one set of quotas since we don't have any env vars:

```suggestion
    retry --wait 1 -n 5 sh -c "tests.nested exec udevadm info -e | NOMATCH ID_MODEL=QEMU_USB_SERIAL"
```